src/main/java/de/wejago/hichi2influx/service/JsonSubscriber.java

```diff
@@ -0,0 +1,81 @@
+package de.wejago.hichi2influx.service;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.influxdb.client.domain.WritePrecision;
+import com.influxdb.client.write.Point;
+import de.wejago.hichi2influx.config.Device;
+import de.wejago.hichi2influx.repository.InfluxDbRepository;
+import java.time.Instant;
+import java.util.Iterator;
+import java.util.Map;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.collections.map.HashedMap;
+import org.eclipse.paho.client.mqttv3.MqttMessage;
+
+@Slf4j
+@RequiredArgsConstructor
+public class JsonSubscriber extends Subscriber{
+    private final ObjectMapper objectMapper;
+    private final InfluxDbRepository influxDbRepository;
+    private final Device device;
+
+    @Override public void messageArrived(String s, MqttMessage mqttMessage) throws Exception {
```

Comment:
What is this s parameter?

Reply:
It is the topic. I changed the parameter name to `topic`.